### PR TITLE
feat(extension): add docsearch

### PR DESCRIPTION
**Pull request by Fatpandac**
## Description

Search documentation in Raycast.
## Screencast

image

## Checklist

- [X] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [X] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [X] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [X] I checked that files in the `assets` folder are used by the extension itself
- [X] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by Fatpandac** [comment]
@danielsequeira Can you take a look?

File: extensions/docsearch/package.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  "icon": "docsearch.png",
+  "author": "Fatpandac",
+  "categories": [
+    "Documentation"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "docsearch",
+      "title": "Search Documentation",
```
Comment (danielsequeira) [inline extensions/docsearch/package.json:15]:
```suggestion
      "title": "Search Documentation",
      "subtitle": "DocSearch",
```

File: extensions/docsearch/src/algolia/APIData.ts [hunk 2]
```diff
@@ -0,0 +1,178 @@
+import type { IAPIData } from "./types";
```
Comment (danielsequeira) [inline extensions/docsearch/src/algolia/APIData.ts:1]: for the sake of consistency it would make more sense to rename this file to "apiData.ts"

**Comment by danielsequeira** [comment]
@Fatpandac done, just 2 tiny comments

**Comment by Fatpandac** [comment]
@danielsequeira OK, thank you so much.

**Review by danielsequeira — APPROVED** [review]

**Pull request merged**